browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -5395,16 +5393,15 @@ index fdc97dc462c87526aa188f7968d60a0c83d680d9..211d323fc7b359ffa3c8431af2bdc1d8
  
      m_frame.document()->statePopped(stateObject ? Ref<SerializedScriptValue> { *stateObject } : SerializedScriptValue::nullValue());
      m_client->dispatchDidPopStateWithinPage();
-@@ -1489,6 +1490,8 @@ void FrameLoader::loadWithNavigationAction(const ResourceRequest& request, Navig
-         return completionHandler();
-     }
+@@ -1481,6 +1482,7 @@ void FrameLoader::load(FrameLoadRequest&& request)
  
+ void FrameLoader::loadWithNavigationAction(const ResourceRequest& request, NavigationAction&& action, FrameLoadType type, RefPtr<FormState>&& formState, AllowNavigationToInvalidURL allowNavigationToInvalidURL, CompletionHandler<void()>&& completionHandler)
+ {
 +    InspectorInstrumentation::frameScheduledNavigation(m_frame, Seconds(0));
```

Comment:
This essentially reverts https://github.com/microsoft/playwright/pull/11234, poor rebase?